etl/steps/data/garden/wb/2025-07-01/income_groups.meta.yml

```diff
@@ -5,16 +5,17 @@ definitions:
         - Economic Growth
     processing_level: major
     description_from_producer: |-
-      For the current 2025 fiscal year, low-income economies are defined as those with a GNI per capita, calculated using the [World Bank Atlas method](https://datahelpdesk.worldbank.org/knowledgebase/articles/378832-what-is-the-world-bank-atlas-method), of $1,145 or less in 2023; lower middle-income economies are those with a GNI per capita between $1,146 and $4,515; upper middle-income economies are those with a GNI per capita between $4,516 and $14,005; high-income economies are those with more than a GNI per capita of $14,005.
+      For the current 2026 fiscal year, low-income economies are defined as those with a GNI per capita, calculated using the [World Bank Atlas method](https://datahelpdesk.worldbank.org/knowledgebase/articles/378832-what-is-the-world-bank-atlas-method), of $1,135 or less in 2024; lower middle-income economies are those with a GNI per capita between $1,136 and $4,495; upper middle-income economies are those with a GNI per capita between $4,496 and $13,935; high-income economies are those with more than a GNI per capita of $13,935.
```

Comment:
In case you think this could make your life easier on the next update, consider using placeholders, e.g. {YEAR_FISCAL}, and then `paths.create_dataset([tb], yaml_params={"YEAR_FISCAL": year_fiscal})` (where `year_fiscal` and other variable fields could be extracted from the metadata. Or the entire `description_from_producer` could be extracted programmatically. Up to you!

Reply:
Yeah, though life's easy with this updates anyway :). As you see, we don't have to change much and it's good to check the thresholds